extensions.json

```
@@ -1094,8 +1094,8 @@
     {
       "id": "mtxr.sqltools",
       "repository": "https://github.com/mtxr/vscode-sqltools",
-      "version": "0.23.0",
-      "checkout": "v0.23.0"
+      "version": "1.0.0",
+      "checkout": "log/v1.0.0"
```

I don't understand the releases for https://github.com/mtxr/vscode-sqltools:
- Latest GitHub Release seems to be [v0.22.11 (Jul 23, 2020)](https://github.com/mtxr/vscode-sqltools/releases/tag/v0.22.11)
- Highest release tag seems to be [v1.2.0 (May 12, 2020)](https://github.com/mtxr/vscode-sqltools/tree/v1.2.0)
- The `package.json` file in `master` has [1.0.0 (Aug 6, 2020)](https://github.com/mtxr/vscode-sqltools/blob/master/package.json#L5)
- The Microsoft VSCode Marketplace currently has [0.23.0 (Aug 21, 2020)](https://marketplace.visualstudio.com/items?itemName=mtxr.sqltools)

Maybe revert this change and stop trying to auto-upgrade.

```suggestion
      "version": "0.23.0",
      "checkout": "v0.23.0"
```

---

Stopped auto-upgrades: https://github.com/open-vsx/publish-extensions/commit/67431d3199e3127d12adae6f5ed035b23e52bbe3